tools/cardinal/combine.xml

```diff
@@ -1,25 +1,25 @@
-<tool id="cardinal_combine" name="MSI combine" version="@VERSION@.0">
+<tool id="cardinal_combine" name="MSI combine" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
     <description>
         combine several mass spectrometry imaging datasets into one
     </description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <expand macro="requirements">
-        <requirement type="package" version="3.3.5">r-ggplot2</requirement>
+        <!--requirement type="package" version="3.3.5">r-ggplot2</requirement>
         <requirement type="package" version="0.12">r-maldiquantforeign</requirement>
-        <requirement type="package" version="1.20">r-maldiquant</requirement>
+        <requirement type="package" version="1.20">r-maldiquant</requirement-->
     </expand>
     <command detect_errors="exit_code">
     <![CDATA[
         #for $i, $infile in enumerate($infiles):
             #if $infile.ext == 'imzml'
-                ln -s '${infile.extra_files_path}/imzml' infile_${i}.imzML &&
-                ln -s '${infile.extra_files_path}/ibd' infile_${i}.ibd &&
+                cp '${infile.extra_files_path}/imzml' infile_${i}.imzML &&
```

Comment:
Is copying really necessary?

Reply:
unfortunately yes, otherwise neither imzml nor analyze75 files will be loaded